pages/projects.tsx

```diff
@@ -68,11 +71,16 @@ export default Projects;
 export const getStaticProps: GetStaticProps<ProjectsProps> = async () => {
   const projects = await getProjects();
   const githubColors = await getGithubColors();
+
+  // await writeJsonFile('./test/fixtures/AllProjects.json', JSON.stringify(projects));
+
+  await writeJsonFile('./test/fixtures/AllProjects.json', projects);
+
   return {
     props: {
       projects,
       githubColors: githubColors,
     },
-    revalidate: 60,
+    revalidate: 86400,
```

Comment:
I'd probably store this as a constant somewhere (showing the math of how to get there like `REVALIDATE_DAILY = 60 * 60 * 60 *24` like in types.ts, and just import that in here, as opposed to just slapping the raw number in there. (fyi, revalidate is in seconds so this is in fact run every 24 minutes instead of every 24 hours haha.) We can actually probably change this to like every 3 hrs instead, I think every day isn't necessary since we're rate limited by the hour.

Reply:
So it should be REVALIDATE_DAILY = 3 * 60 * 60, since 3 hours, 60 minutes per hour, 60 seconds per minute